src/sfdxFlags.ts

```diff
@@ -648,7 +648,7 @@ function validateCustomFlag<T>(key: string, flag: flags.Any<T>): flags.Any<T> {
   return flag;
 }
 
-function isBuiltin(flag: object): flag is flags.Builtin {
+function isBuiltin(flag: Record<string, unknown>): flag is flags.Builtin {
```

Comment:
this makes me super nervous

Reply:
will revert and ts-comment it.  It's not public, but I can see something untyped getting passed down into it.